estimagic/optimization/reparametrize.py

```diff
@@ -50,13 +60,34 @@ def reparametrize_to_internal(params, constraints):
         internal.loc[actually_fixed, "__fixed__"] = True
 
     internal = internal.query("~__fixed__")
+    internal.drop(["__fixed__"], axis=1, inplace=True)
```

Comment:
Since you are using pandas >= 0.24.0 you can use ``internal.drop(columns="__fixed__", inplace=True)``

Reply:
done.